docs/faq.md

```diff
@@ -16,21 +16,19 @@ The Dusty daemon, which runs as root, manages your GitHub repos by default.
 To get permissions to clone from GitHub, the
 Daemon will make use of your unprivileged user's SSH_AUTH_SOCK. This means that the daemon's
 behavior using git should be the same as your user's, in terms of SSH authentication.
+(See [this GitHub help article](https://help.github.com/articles/generating-ssh-keys/)
+for some info about setting up GitHub ssh keys.)
```

Comment:
you are right.  This would cause confusion.  Should have looked. will remove.  talking like an idiot here. cannot stop. help. me. i. am. lost